devops/cloudformation.json

```diff
@@ -0,0 +1,122 @@
+{
+  "Parameters": {
+    "NTTHostname": {
+      "Type": "String",
+      "Default": "ntt-beta.labs.transitmatters.org",
+      "AllowedPattern": "^traintracker\\.transitmatters\\.org$|^.*\\.labs\\.transitmatters\\.org$",
+      "Description": "The hostname for the site; for example: traintracker.transitmatters.org."
+    },
+    "NTTDomain": {
+      "Type": "String",
+      "Default": "labs.transitmatters.org",
+      "AllowedPattern": "^traintracker\\.transitmatters\\.org$|^labs\\.transitmatters\\.org$",
+      "Description": "The parent domain of the site. Most likely traintracker.transitmatters.org or labs.transitmatters.org."
+    },
+    "NTTCertArn": {
+      "Type": "String",
+      "Description": "The ACM ARN of the SSL certificate."
+    }
+  },
+  "Resources": {
+    "NTTInstance": {
+      "Type": "AWS::EC2::Instance",
+      "Properties": {
+        "ImageId": "ami-070650c005cce4203",
```

Comment:
Does this image change often? If so, do we have a workflow to automatically update and upload them?